### PR TITLE
SISRP-17261 - UX Develop: Dashboard (faculty) - Faculty Resources

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-17261

### CODE REVIEW

File: src/assets/templates/academics_classinfo.html [hunk 1]
```diff
@@ -153,6 +153,30 @@ <h3 class="cc-book-title" data-ng-bind="sectionBookList.sectionLabel" data-ng-if
           </div>
         </div>
       </div>
+
+      <div class="cc-widget">
```
Comment (christianvuerings) [inline src/assets/templates/academics_classinfo.html]: @dfkaye Rachel just told me we should only show this to faculty so we should probably hide this behind an `ng-if`.
Also, could we put it in a separate template?

Comment (dfkaye) [inline src/assets/templates/academics_classinfo.html]: @christianvuerings ~ what location do you suggest for the new template? would `templates/widgets/academics/faculty-resources.html` be OK?

Comment (dfkaye) [inline src/assets/templates/academics_classinfo.html]: @christianvuerings ~ nm - I'll put it at `templates/faculty-resources.html` for now...

File: src/assets/templates/faculty-resources.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <h3>Classes and Schedule</h3>
+    <ul class="cc-list-bullets">
+      <li><a href="http://schedule.berkeley.edu">Schedule of Classes</a></li>
+      <li><a href="http://guide.berkeley.edu">Class Catalog:</a> Berkeley Academic Guide</li>
+    </ul>
+
+    <h3>Other</h3>
+    <ul class="cc-list-bullets">
+      <li><a href="https://www.ets.berkeley.edu/services-facilities/assistive-technology">Assistive Technology:</a> <abbr title="Americans with Disabilities Act">ADA</abbr> Support</li>
+      <li><a href="https://www.ets.berkeley.edu/services-facilities/bcourses">bCourses:</a> Getting started and resources</li>              <li><a href="https://www.ets.berkeley.edu/services-facilities/clickers">Clickers</a></li>
```
Comment (christianvuerings) [inline src/assets/templates/faculty-resources.html]: @dfkaye any reason why this `<li><a href="https://www.ets.berkeley.edu/services-facilities/clickers">Clickers</a></li>` is on the same line?

Comment (dfkaye) [inline src/assets/templates/faculty-resources.html]: @christianvuerings ~ fixed

**Pull request merged**